hours/models.py

```diff
@@ -258,6 +270,16 @@ def get_daily_opening_hours(self, start_date, end_date):
 
         return processed_opening_hours
 
+    def get_organizations(self):
+        """Returns the resources and its parents organizations"""
+        organizations = set()
+
+        if self.organization:
+            organizations.add(self.organization)
+            organizations.update(_get_all_parent_organizations(self))
```

Comment:
Probably we should get all parents even if the resource has no organization? Currently the subsections have no organization set, unless we want to change that.

Reply:
Good point. Fixed.